index.html

```diff
@@ -928,7 +929,8 @@
           Interface <a><code>PresentationRequest</code></a>
         </h3>
         <pre class="idl">
-          [Constructor(DOMString url)]
+          [Constructor(DOMString url),
+           Constructor(DOMString[] urls)]
```

Comment:
Do we want to allow (DOMString[] or DOMString) for simplicity and backward compatibility?